packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -93,7 +94,7 @@ function preload(
     // in that case fallback to getAttribute
     const cspNonce = cspNonceMeta?.nonce || cspNonceMeta?.getAttribute('nonce')
 
-    promise = Promise.all(
+    promise = Promise.allSettled(
```

Comment:
I confirmed that `Promise.allSettled` is also supported by our default modern browser target.
https://caniuse.com/mdn-javascript_builtins_promise_allsettled